releases/v19.2.0-beta.20190910.md

```diff
@@ -153,26 +164,272 @@ $ docker pull cockroachdb/cockroach-unstable:v19.2.0-beta-20190903
 
 ### Contributors
 
-This release includes 382 merged PRs by 38+ authors.
+This release includes 1570 merged PRs by 67 authors.
 We would like to thank the following contributors from the CockroachDB community:
 
-- Arseni Lapunov
-- Elliot Courant
+- Arseni Lapunov (first-time contributor)
+- Arul Ajmani (first-time contributor)
+- Elliot Courant (first-time contributor)
+- George Papadrosou
+- Georgia Hong (first-time contributor)
+- HyeonGyu Lee (Vazrupe) (first-time contributor)
 - Jaewan Park
+- Joel Kenny (first-time contributor)
```

Comment:
We should remove Cockroach Labs engineers. The script usually catches this but obviously doesn't have everyone. I think Joe, Spas, and Irfan are the ones to remove, but I might be missing someone.